frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -303,14 +306,6 @@ function translateLogoHeight(theme: any, size: string): string {
   return theme.sizes.defaultLogoHeight
 }
 
-/**
- * The width of the scrollbar gutter is estimated to be 8px.
- *
- * Note that the actual width of the scrollbar gutter is browser and OS
- * dependent.
- */
-const SCROLLBAR_GUTTER_WIDTH_ESTIMATE = "8px"
```

Comment:
No need to estimate the gutter width anymore since we calculate it and store as `--scrollbar-width`